source/module_basis/module_ao/ORB_table_phi.cpp

```diff
@@ -170,6 +170,21 @@ void ORB_table_phi::cal_ST_Phi12_R
 			{
 				k1_dot_k2[ik] = n1.getPsi_k2(ik) * n2.getPsi_k2(ik);
 			}
+			
```

Comment:
please add unittest with gtest for function "ORB_table_phi::cal_ST_Phi12_R" before you modified it.